### DISCUSSION TITLE
Search in descriptions, tagging

### DISCUSSION THREAD

**Discussion by vladobb** [immich-app/immich#14537] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi friends, I read some post from APR24 about search in descriptions, I installed IMMICH recently and this function is not available yet. Can you please let me  know thcurrent e status? As a volunteer can I help in any way? Because I really do not understand why to play extensively with some kind of machine learning models etc. if I am not able to find photo based on simple tag.

Thanks a lot

V

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Dupe of #7823. This is being worked on in #14380
